src/nwb_datajoint/spikesorting/spikesorting_curation.py

```diff
@@ -381,17 +401,18 @@ def _compute_metric(self, waveform_extractor, metric_name, **metric_params):
         metric_func = _metric_name_to_func[metric_name]
         # TODO clean up code below
         if metric_name == 'isi_violation':
-            metric = metric_func([], waveform_extractor, **metric_params)
-        elif metric_name == 'snr':
+            metric = metric_func(waveform_extractor, **metric_params)
+        elif (metric_name == 'snr' or
+              metric_name == 'peak_offset'):
```

Comment:
what if `metric_params` doesn't contain a key called `peak_sign`? do the lines below that reference `metric_params['peak_sign']` not throw an error?

Reply:
good point; I'll fix that